migrations/0016_drop_unused_mission_and_goal_fields.py

```diff
@@ -0,0 +1,26 @@
+"""
+drop unused mission and goal fields
+date created: 2019-05-04 21:44:40.156692
+"""
+
+def add_foreign_key(migrator, from_table, from_field, to_table, to_field, **kwargs):
+  """Add a foreign key, which playhouse-moves doesn't seem to do properly."""
```

Comment:
huh that's funny. how did it work in the last migration that added the key constraints?

Reply:
I used the same helper function last time, after reading the peewee-moves code and discovering this didn't work. I tried for 30 minutes to reuse the same function somehow, including an insane attempt to make relative imports work for things beginning with numbers, but thought better of it and gave up. peewee-moves is not my favorite library.